config/locales/pt-BR/views.yml

```diff
@@ -56,6 +56,18 @@ pt-BR:
         E avisem hora extra para <strong>"rh@codeminer42.com"</strong>
       </p>
       Abs
+    notify_fill_contribution_description_html: |
+      <p>
+        Olá,
+      </p>
+      <p>
+        Você possui a seguinte contribuição aguardando descrição da PR.
+      </p>
+      <p>
+        Acessar PR&#58; 
+        <a href='https://%{host}/contributions/%{contribution.id}'>%{contribution.link}</a>
```

Comment:
you could pass the url as param, so we will keep using the URL helpers

Reply:
Done